psycop/common/model_training_v2/problem_type/eval_dataset_base.py

```diff
@@ -4,27 +4,41 @@
 
 import polars as pl
 
-from psycop.common.model_training_v2.metrics.binary_metrics.base import BinaryMetric
-
-from ..presplit_preprocessing.polars_frame import PolarsFrame
+from psycop.common.global_utils.pickle import write_to_pickle
+from psycop.common.model_training_v2.metrics.base_metric import CalculatedMetric
+from psycop.common.model_training_v2.metrics.binary_metrics.base_binary_metric import (
+    BinaryMetric,
+)
 
 
 @dataclass
 class BaseEvalDataset:
-    pred_time_uuids: pl.Expr
-    y_hat_probs: pl.Expr
-    y: pl.Expr
-    df: PolarsFrame
+    pred_time_uuids: str
+    y_hat_probs: str
+    y: str
+    df: pl.DataFrame
```

Comment:
If they are pl.Expr, they can be pl.Col("pred_time_uuids") for example. Might want a comment for that instead?

Reply:
There doesn't seem to be a nice way of selecting columns using `pl.col` though -- if we use `.select` we get a type error in `metric.calculate` because it expects a series. Using `str` seems to me the most flexible solution at the moment, but open to suggestions